### ISSUE TITLE
Focus ring does not adhere to theme colour

### ISSUE THREAD

**Issue by Alwigi** [mkocansey/bladewind#442]
When setting the primary and secondary colours in tailwind.confg.js as below

    theme: {
        extend: {
            colors: {
                primary: colors.amber,
                secondary: colors.cyan,
              }
        },
    },

buttons inherit the colour as expected, but the focus ring on text input does not (blue ring should be amber on the input below?).
I have noticed the same on dropdowns, datepicker, not sure which others will have this behaviour

![Image](https://github.com/user-attachments/assets/4e65cec9-e075-46ac-a373-9e26fd20f8a8)

Is this a bug or by design?

**Comment by mkocansey** [comment]
@Alwigi This currently by design but now that you raise this, I'm wondering if all controls should just go with the primary colours 🤔. Will add this and make it configurable.
